bridges/snowbridge/primitives/inbound-queue/src/v2/converter.rs

```diff
@@ -241,6 +241,8 @@ where
 					.encode()
 					.into(),
 			},
+			// Deposit leftover funds to Snowbridge sovereign
+			DepositAsset { assets: Wild(AllCounted(2)), beneficiary: bridge_owner.clone().into() },
```

Comment:
The one problem with removing the excess fee handling from the converter is that for the CreateAsset case, the UX cannot specify the fee refund instruction (since it does not send raw XCM). I have added a RefundSurplus and DepositAsset to deposit any leftover fees to the Snowbridge sovereign. What do you think of this? Maybe we should deposit to the claimer (if present), otherwise Snowbridge sovereign?

Reply:
> Maybe we should deposit to the claimer (if present), otherwise Snowbridge sovereign?

I prefer this one.

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/32/commits/c1484452b34459d6fbad7dfc4577a297397d03e6 and https://github.com/vgeddes/polkadot-sdk/pull/32/commits/db80fd5148ff1adc99a81821b3438b01af2070cc